docs/workflow/testing/libraries/testing.md

```diff
@@ -1,54 +1,81 @@
 # Testing Libraries
 
-We use the OSS testing framework [xunit](https://github.com/xunit/xunit).
+## Full Build and Test Run
 
-To build the tests and run them you can call the libraries build script. For libraries tests to work, you must have built the coreclr or mono runtime for them to run on.
+These example commands automate the test run and all pre-requisite build steps in a single command from a clean enlistment.
 
-**Examples**
-- The following shows how to build only the tests but not run them:
+- Build all pre-requisites (clr + libs) and tests and run them using CoreCLR:
 ```
-build.cmd/sh -subset libs.tests
+build.cmd/sh -subset clr+libs+libs.tests -test
 ```
 
-- The following builds and runs all tests using clr:
+- Build all pre-requisites (mono + libs) and tests and run them using Mono:
 ```
-build.cmd/sh -subset clr+libs.tests -test
+build.cmd/sh -subset mono+libs+libs.tests -test
 ```
 
-- The following builds and runs all tests using mono:
+- Run all tests - Builds clr in release, libs+tests in debug:
 ```
-build.cmd/sh -subset mono+libs.tests -test
+build.cmd/sh -subset clr+libs+libs.tests -test -rc Release
 ```
 
-- The following builds and runs all tests in release configuration:
+- Run all tests - Build mono and libs for x86 architecture:
 ```
-build.cmd/sh -subset libs.tests -test -c Release
+build.cmd/sh -subset mono+libs+libs.tests -test -arch x86
 ```
 
-- The following builds clr in release, libs in debug and runs all tests:
+## Partial Build and Test Runs
+
+Doing full build and test runs takes a long time and is very inefficient if you need to iterate on a change.
+For greater control and efficiency individual parts of the build + testing workflow can be run in isolation. 
+See the [Building instructions](../../building/libraries/README.md) for more info on build options.
+
+### Test Run Pre-requisites
+Before any tests can run we need a complete build to run them on. Specifically:
+1. Build a runtime (clr or mono)
 ```
-build.cmd/sh -subset clr+libs+libs.tests -test -rc Release
+build.cmd/sh -subset clr
+```
+OR
+```
+build.cmd/sh -subset mono
+```
+2. Build all the libraries
+```
+build.cmd/sh -subset libs
 ```
 
-- The following builds mono and libs for x86 architecture and runs all tests:
+Building the `libs` subset automatically copies all product binaries into a testhost folder in the bin directory.
```

Comment:
I restructured the content on the page to hopefully put this deployment step into better context. Let me know if this still appears to be surfacing unnecessary detail.